crates/volta-core/src/project/mod.rs

```diff
@@ -177,7 +177,11 @@ impl Project {
         self.platform()
             .map_or(false, |platform| platform.yarn.is_some())
             && self.manifest_file.parent().map_or(false, |base_dir| {
-                base_dir.join(".pnp.js").exists() || base_dir.join(".pnp.cjs").exists()
+                base_dir.join(".pnp.js").exists()
+                    || base_dir.join(".pnp.cjs").exists()
+                    || self
+                        .workspace_roots()
```

Comment:
I think this winds up duplicating work: `self.workspace_roots()` includes as the first entry the parent of `self.manifest_file`, so we're re-checking that. Instead, I think this check that you have is already exactly what we want (and can replace the whole `self.manifest_file...` check)

```rust
self
    .workspace_roots()
    .any(|dir| dir.join(".pnp.cjs").exists() || dir.join(".pnp.js").exists())
```

Reply:
Thanks I applied changes, will wait until #1388 is merged to resolve conflicts.

Reply:
Looks like #1388 is merged, can we merge this now?